src/components/common/chat/ChatBox.tsx

```diff
@@ -13,19 +13,20 @@ import {
 import { DEFAULT_INITIAL_PROMPT } from '../../../config/appSettings';
 import {
   ANONYMOUS_USER,
-  CHATBOT_PREFIX,
-  CHATBOT_RESPONSE_URL,
   MAX_CONVERSATION_LENGTH,
   MAX_CONVERSATION_LENGTH_ALERT,
   SCROLL_SAFETY_MARGIN,
-  STUDENT_PREFIX,
 } from '../../../config/constants';
+import { CHAT_BOT_ERROR_MESSAGE } from '../../../config/messages';
+import { mutations } from '../../../config/queryClient';
 import { CHATBOT_MODE_CY, messagesDataCy } from '../../../config/selectors';
 import {
   DEFAULT_BORDER_RADIUS,
   LIGHT_GRAY,
   LIGHT_VIOLET,
 } from '../../../config/stylingConstants';
+import { ThreadMessage } from '../../../interfaces/chatbot';
+import { buildPrompt } from '../../../utils/chatbot';
```

Comment:
This looks like it is used in all the apps that use the chatbot, should it be exported as a util function from somewhere common ?